JotunnLib/Utils/ModCompatibility/ModCompatibility.cs

```diff
@@ -30,6 +31,15 @@ internal static void Init()
             Main.Harmony.PatchAll(typeof(ModCompatibility));
         }
 
+        public static bool IsJotunnOnServer()
+        {
+            if (ZNet.instance && ZNet.instance.IsClientInstance())
+            {
+                return LastServerVersion != null;
```

Comment:
All right, I'll look over the code and see what I can come up with. Currently none of the `ModuleVersionData` appear to be cached so it could take me a bit to figure out which instance to check against and ensure that it stays current.